netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -679,14 +731,6 @@ public TlsProtocolNegotiatorClientFactory(SslContext sslContext) {
     }
 
     @Override public ProtocolNegotiator newNegotiator() {
-      SslContext sslContext = this.sslContext;
-      if (sslContext == null) {
-        try {
-          sslContext = GrpcSslContexts.forClient().build();
-        } catch (SSLException ex) {
-          throw new RuntimeException(ex);
-        }
-      }
       return tls(sslContext);
```

Comment:
Based on this change, should you have the `checkNotNull` check for `sslContext` on line 730?